C/include/simplicity/elements.h

```diff
@@ -88,4 +88,24 @@ typedef struct transaction transaction;
  * Precondition: NULL != rawTx
  */
 extern transaction* mallocTransaction(const rawTransaction* rawTx);
+
+/* Deserialize a Simplicity program from 'file' and execute it in the environment of the 'ix'th input of 'tx'.
+ * If the file isn't a proper encoding of a Simplicity program, '*success' is set to false.
+ * If EOF isn't encountered at the end of decoding, '*success' is set to false.
+ * If 'cmr != NULL' and the commitment Merkle root of the decoded expression doesn't match '*wmr' then '*success' is set to false.
```

Comment:
Typo `*wmr` should be `*cmr`

I also think writing `*cmr` to mean "the 8 words pointed at by `cmr`" is a bit confusing, but I'm not sure what a better notation would be.

Reply:
Done.